### PR TITLE
Create v1.1.1.md

### PR DESCRIPTION
**Pull request by bk201**
Co-authored-by: Rebecca <zhuo.rebecca@gmail.com>

### CODE REVIEW

File: v1.1.1.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+:white_check_mark: [harvester-v1.1.1-amd64.sha512](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-amd64.sha512)
+:memo:  [harvester-v1.1.1-version.yaml](https://releases.rancher.com/harvester/v1.1.1/version.yaml)
+
+# Install
+Harvester can be installed with the [ISO](https://docs.harvesterhci.io/v1.1/install/iso-install/) image or via [PXE](https://docs.harvesterhci.io/v1.1/install/pxe-boot-install/) boot.
+
+# Highlights
+* Installer crash after input hostname [[#2937](https://github.com/harvester/harvester/issues/2937)]
+* Host IP is inconsistent [[#2962](https://github.com/harvester/harvester/issues/2962)]
+* Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface [[#3045](https://github.com/harvester/harvester/issues/3045)]
```
Comment (guangbochen) [inline v1.1.1.md:23]:
```suggestion
* Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface [[#3045](https://github.com/harvester/harvester/issues/3045)]
* Host IP is inconsistent [[#2962](https://github.com/harvester/harvester/issues/2962)]
* Installer crash after input hostname [[#2937](https://github.com/harvester/harvester/issues/2937)]

```

File: v1.1.1.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
```
Comment (guangbochen) [inline v1.1.1.md:27]: do we require to shut down all VMs upgraded from v1.1.0?

File: v1.1.1.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+* [Support upgrade from v1.0.3 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
+
+# Major Bug Fixes
+* Cant's display monitoring dashboard after upgrade, alertmanager, prometheus and grafana monitoring pods Terminating [[#2984](https://github.com/harvester/harvester/issues/2984)]
+* migrate_harv_mgmt_to_mgmt_br.sh should remove ClusterNetwork resource [[#3047](https://github.com/harvester/harvester/issues/3047)]
```
Comment (guangbochen) [inline v1.1.1.md:33]:
```suggestion
* `migrate_harv_mgmt_to_mgmt_br.sh` should remove ClusterNetwork resource [[#3047](https://github.com/harvester/harvester/issues/3047)]
```

File: v1.1.1.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+# Dependency Component Versions
+| Component | Version |
+| ------ | ---------|
+| Longhorn | [v1.3.2](https://github.com/longhorn/longhorn/releases/tag/v1.3.2) |
+| KubeVirt | [v0.54.0](https://github.com/kubevirt/kubevirt/releases/tag/v0.54.0) |
+| Rancher | [v2.6.9](https://github.com/rancher/rancher/releases/tag/v2.6.9) |
+| RKE2 | [v1.24.7+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.24.7%2Brke2r1) |
+
+# Contributors
+Thanks to all the contributors that make the Harvester v1.1.0 release possible!
```
Comment (guangbochen) [inline v1.1.1.md:63]:
```suggestion
Thanks to all the contributors that make the Harvester v1.1.1 release possible!
```

File: v1.1.1.md (hunk 2 above)
Comment (bk201) [reply v1.1.1.md]: No need. fixed!

**Review by guangbochen — APPROVED** [review]
LGTM and please squash the commits before the merge, thanks.

File: v1.1.1.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+* GUI change: `Network` is renamed to `VM Networks` [[3088](https://github.com/harvester/harvester/issues/3088)]
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+* [Support upgrade from v1.0.3 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * We highly recommend upgrading v1.0.3 clusters to v1.1.1 (skip version v1.1.0).
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
+
+# Major Bug Fixes
+* Cant's display monitoring dashboard after upgrade, alertmanager, prometheus and grafana monitoring pods Terminating [[#2984](https://github.com/harvester/harvester/issues/2984)]
```
Comment (yasker) [inline v1.1.1.md:39]: `Can't`

File: v1.1.1.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+# Known Issues
+* Windows VMs crashing [[#2522](https://github.com/harvester/harvester/issues/2522)]
+* Unable to provision RKE2 cluster from external Harvester cloud credential [[#2590](https://github.com/harvester/harvester/issues/2590)] (**Deprecated**)
+* Can't completely install Harvester cloud provider on K3s v1.24.4 and v1.23.10 cluster [[#2998](https://github.com/harvester/harvester/issues/2998)]
+* Upgrade
+  * [Multiple nodes pre-drains in an upgrade[3216](https://github.com/harvester/harvester/issues/3216)]
+  * Upgrade stop at upgrading node3 stage, stuck in Pre-drained status [[#3021](https://github.com/harvester/harvester/issues/3021)]
+  * longhorn-system can't be evicted [[#2053](https://github.com/harvester/harvester/issues/2053)]
+  * Upgrade is stuck in "Pre-drained" state (Volume has too many system snapshots) [[#2640](https://github.com/harvester/harvester/issues/2640)]
+  * Upgrade stuck in upgrading first node: Job was active longer than specified deadline [[#2894](https://github.com/harvester/harvester/issues/2894)]
```
Comment (yasker) [inline v1.1.1.md:59]: Do we have workarounds documented for known issues? Need the links here as well.

File: v1.1.1.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+* Tag rc version, harvester plugin needs to build the corresponding package [[#3145](https://github.com/harvester/harvester/issues/3145)]
+* Upgrade: finished upgrades interfere with newer ones [[#3152](https://github.com/harvester/harvester/issues/3152)]
+* The NICs should keep up after deleting the VLAN config [[#3166](https://github.com/harvester/harvester/issues/3166)]
+* Incomplete dependency chart installation after upgrading 1.0.3 to 1.1.0 and master-head. [[#3168](https://github.com/harvester/harvester/issues/3168)]
+* Upgrade: post-drain job fails when upgrading to the same version [[#3175](https://github.com/harvester/harvester/issues/3175)]
+* Cannot use the secret for all outputs [[#3184](https://github.com/harvester/harvester/issues/3184)]
+
+# Known Issues
+* Windows VMs crashing [[#2522](https://github.com/harvester/harvester/issues/2522)]
+* Unable to provision RKE2 cluster from external Harvester cloud credential [[#2590](https://github.com/harvester/harvester/issues/2590)] (**Deprecated**)
```
Comment (yasker) [inline v1.1.1.md:52]: Put this one at the last and starting with `Deprecated`.

And, have we decided to not support it in the future? It doesn't indicate in the issue and still marked for v1.1.2.

File: v1.1.1.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+* Can't completely install Harvester cloud provider on K3s v1.24.4 and v1.23.10 cluster [[#2998](https://github.com/harvester/harvester/issues/2998)]
+* Upgrade
+  * [Multiple nodes pre-drains in an upgrade[3216](https://github.com/harvester/harvester/issues/3216)]
```
Comment (yasker) [inline v1.1.1.md:55]: The formatting is wrong.

File: v1.1.1.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+ 
+
+# Highlights
+* Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface [[#3045](https://github.com/harvester/harvester/issues/3045)]
+* Host IP is inconsistent [[#2962](https://github.com/harvester/harvester/issues/2962)]
+* Installer crash after input hostname [[#2937](https://github.com/harvester/harvester/issues/2937)]
+* GUI change: `Network` is renamed to `VM Networks` [[3088](https://github.com/harvester/harvester/issues/3088)]
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
```
Comment (yasker) [inline v1.1.1.md:33]: Add
`Only the following upgrade path is supported.`.

File: v1.1.1.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+* [Support upgrade from v1.0.3 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * We highly recommend upgrading v1.0.3 clusters to v1.1.1 (skip version v1.1.0).
```
Comment (yasker) [inline v1.1.1.md:35]: Remove two indents here. 

Move the two statements to the top of the section, and briefly explain why skipping version v1.1.0. Those two statements are the more critical part of the `Upgrade` section. They need to be front and center.

**Review by yasker — COMMENTED** [review]
Do we have backward compatibility changes? Even we don't have any new changes for that section, we should mention there is `not` to confirm.

Also, can we mention the backward compatibility (`behavior changes`?) for v1.1.0 here as well? Or it's better to link to the doc. Since we don't have it for 1.1.0 release notes.

**Comment by yasker** [comment]
@btat Can you help to proofread this?

File: v1.1.1.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+# Known Issues
+* Windows VMs crashing [[#2522](https://github.com/harvester/harvester/issues/2522)]
+* Unable to provision RKE2 cluster from external Harvester cloud credential [[#2590](https://github.com/harvester/harvester/issues/2590)] (**Deprecated**)
+* Can't completely install Harvester cloud provider on K3s v1.24.4 and v1.23.10 cluster [[#2998](https://github.com/harvester/harvester/issues/2998)]
+* Upgrade
+  * [Multiple nodes pre-drains in an upgrade[3216](https://github.com/harvester/harvester/issues/3216)]
+  * Upgrade stop at upgrading node3 stage, stuck in Pre-drained status [[#3021](https://github.com/harvester/harvester/issues/3021)]
+  * longhorn-system can't be evicted [[#2053](https://github.com/harvester/harvester/issues/2053)]
+  * Upgrade is stuck in "Pre-drained" state (Volume has too many system snapshots) [[#2640](https://github.com/harvester/harvester/issues/2640)]
```
Comment (btat) [inline v1.1.1.md:58]: Could we write/phrase `Pre-drained state/status` the same way for #3021 and #2640. A consistent format will help readers understand it's the same thing being referred to.

File: v1.1.1.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
+
+# Major Bug Fixes
+* Cant's display monitoring dashboard after upgrade, alertmanager, prometheus and grafana monitoring pods Terminating [[#2984](https://github.com/harvester/harvester/issues/2984)]
+* `migrate_harv_mgmt_to_mgmt_br.sh` should remove ClusterNetwork resource [[#3047](https://github.com/harvester/harvester/issues/3047)]
+* Need retry logic around curl in detect_repo [[#3079](https://github.com/harvester/harvester/issues/3079)]
+* Configuring the containerd registry doesn't work anymore during installation [[#3116](https://github.com/harvester/harvester/issues/3116)]
+* Tag rc version, harvester plugin needs to build the corresponding package [[#3145](https://github.com/harvester/harvester/issues/3145)]
+* Upgrade: finished upgrades interfere with newer ones [[#3152](https://github.com/harvester/harvester/issues/3152)]
+* The NICs should keep up after deleting the VLAN config [[#3166](https://github.com/harvester/harvester/issues/3166)]
```
Comment (btat) [inline v1.1.1.md:45]:
```suggestion
* The NICs should remain up after deleting the VLAN config [[#3166](https://github.com/harvester/harvester/issues/3166)]
```

File: v1.1.1.md (hunk 7 above)
Comment (guangbochen) [reply v1.1.1.md]: Yes, and we have planned to remove the UI in v1.1.2 completely.

File: v1.1.1.md [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+* Installer crash after input hostname [[#2937](https://github.com/harvester/harvester/issues/2937)]
+* GUI change: `Network` is renamed to `VM Networks` [[3088](https://github.com/harvester/harvester/issues/3088)]
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+* [Support upgrade from v1.0.3 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * We highly recommend upgrading v1.0.3 clusters to v1.1.1 (skip version v1.1.0).
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
+
+# Major Bug Fixes
```
Comment (rebeccazzzz) [inline v1.1.1.md]:
```suggestion
# Major Bug Fixes
* Grub boot stanza doesn't exclude floppy. Could result in grub hangs. [[#2115](https://github.com/harvester/harvester/issues/2115)]
* Unable to enable passthrough [[#2973](https://github.com/harvester/harvester/issues/2973)]
```

**Review by rebeccazzzz — COMMENTED** [review]
Suggestion to add issues #2115 and #2973 to the major bug fixes section. These issues were closed after the initial release note draft was created. Suggestion to double check the version numbers in the "Dependency Component Versions" section. Suggestion to add a "Behavior Changes" section to call out any behavior changes from v1.0.3 to v1.1.1 since we allow upgrade from v1.0.3 to v1.1.1 for this release.

**Pull request merged**